### DISCUSSION TITLE
Change TI LAUNCHXL-CC1352P-2 for a TI LAUNCHXL-CC1352P-7

### DISCUSSION THREAD

**Discussion by SargonofAssyria** [Koenkk/Z-Stack-firmware#452] [General]
Because of the memory size I would like to switch from CC1352P-2 to a CC1352P-7.
Does anybody know if I have to setup my network all over again or can it be done by switching boards?

**Comment by Koenkk** [comment]
You can just switch the boards (z2m backup functionality will automatically restore everything). Make sure to copy the ieeAddr (write CC1352P-2 ieeeAddr as  CC1352P-7 second ieeeAddr when flashing).

**Comment by SargonofAssyria** [comment]
I suppose that can only be done by CCStudio IDE and compile the bin myself? Or is the an other way?
What is the easy way to find the ieeeAddr from my current adaptor? (FOUND IT in the GUI)

Edit: Found a possible way to change the ieeeAddr with ZigStartGW-MT program, will have to test it.
Did not find the file in CCStudio where to change it.

**Reply by Koenkk** [reply]
This can be done in your flashing tool (CC2538-bsl, zigstart flahser, UniFlash)

**Comment by SargonofAssyria** [comment]
Oke, tnx, Will look into it.
